docs/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ title: Changelog
 
 ## main
 
+* Place all generator options under `config.generate` namespace.
```

Comment:
Is this a breaking change? I'm wondering if we'd want to do a deprecation cycle for these updates where both the old and new options work.

Reply:
Yes, definitely a breaking change that requires a first release with a deprecation warning. And probably wait for the next major version to enforce the new syntax?

Reply:
Expected this — I'll make sure it's done before this is good to go. AFAIK this is the last blocking change that's been identified, so I'll re-request review from @Spone to be sure it's done correctly.

Reply:
Added these in d2e27655376cb30b787cafc4a759e102fbf207e3.